server/routes/topic_page.py

```diff
@@ -67,12 +71,24 @@ def topic_page(topic_id=None, place_dcid=None):
 
   # Find the config for the topic & place.
   topic_place_config = None
-  for config in topic_configs:
-    if place_dcid in config.metadata.place_dcid:
-      topic_place_config = config
-      break
-  if not topic_place_config:
-    return "Error: no config found"
+  if topic_id == _SDG_TOPIC:
+    topic_place_config = topic_configs[0]
+    topic_place_config.metadata.place_dcid.append(place_dcid)
+    for category in topic_place_config.categories:
```

Comment:
maybe add comments that this is to populate all the tiles with comparison places